packages/flutter/lib/src/cupertino/tab_view.dart

```diff
@@ -109,13 +112,17 @@ class CupertinoTabView extends StatelessWidget {
   Route<dynamic> _onGenerateRoute(RouteSettings settings) {
     final String name = settings.name;
     WidgetBuilder routeBuilder;
-    if (name == Navigator.defaultRouteName && builder != null)
+    String title;
+    if (name == Navigator.defaultRouteName && builder != null) {
       routeBuilder = builder;
+      title = defaultTitle;
```

Comment:
I couldn't quite tell from your PR. Is the convenience home: argument on MaterialApp etc going away and everything's going to be specified via Page or wtv we're going to end up calling it or is there still one exception dead-simple API for providing a home widget (to associate a default title with)?

Reply:
all the major stuff (including home) will be backwards compatible, though i am intending there to be an alternative to them that doesn't require that you use them even in the simple case.